### ISSUE TITLE
Game launch and return to ES callbacks?

### ISSUE THREAD

**Issue by scottnath** [Aloshi/EmulationStation#302]
## tl;dr

Are there callbacks or triggers that I can use for when a game is launched from Emulation Station and for when a user returns to ES from a game?
## My setup
- [RetroPie-Setup](https://github.com/petrockblog/RetroPie-Setup)
- rasberry pi b+
- x-arcade dual joystick (x 2)
## Why I need the callback

On game launch, I want to trigger some RGB buttons to change color according to the game's specific controller set up. I would also like to trigger a button map to show the game's mapping (on a second screen).
## Is this a RetroPie job?

Is this actually something that should be programmed into RetroPie and not Emulation Station?

thank you,
Scott
